### DISCUSSION TITLE
Sony X-OCN Support

### DISCUSSION THREAD

**Discussion by goforq** [videovillage/community#165] [Ideas]
Would be amazing to be able to play back Sony RAW files in Screen. Work a lot with Sony Venice 1/2 and would love to get support for them. Thank you hope its possible

**Comment by gregcotten** [comment]
Sony has a hefty yearly licensing fee unfortunately. Likely never to happen unless Sony makes it free.

**Reply by goforq** [reply]
Ugh @Sony 😡

**Reply by tanner-field** [reply]
What a shame. Really would love X-OCN LT support for Burano playback.
